prow/scripts/provision-vm-and-start-kyma.sh

```diff
@@ -28,34 +24,31 @@ else
     LABELS=(--labels "pull-number=$PULL_NUMBER,job-name=kyma-integration")
 fi
 
-echo "
-################################################################################
-# Creating a new instance named kyma-integration-test-${RANDOM_ID}
-################################################################################
-"
-
-gcloud compute instances create "kyma-integration-test-${RANDOM_ID}" \
-    --metadata enable-oslogin=TRUE --image debian-9-stretch-v20181011 \
-    --image-project debian-cloud --machine-type n1-standard-4 --boot-disk-size 20 "${LABELS[@]}"
+ZONE_LIMIT=${ZONE_LIMIT:-5}
+EU_ZONES=$(gcloud compute zones list --filter="name~europe" --limit="${ZONE_LIMIT}" | tail -n +2 | awk '{print $1}')
+
+for ZONE in ${EU_ZONES}; do
+    shout "Attempting to create a new instance named kyma-integration-test-${RANDOM_ID} in zone ${ZONE}"
+    gcloud compute instances create "kyma-integration-test-${RANDOM_ID}" \
+        --metadata enable-oslogin=TRUE \
+        --image debian-9-stretch-v20181011 \
+        --image-project debian-cloud \
+        --machine-type n1-standard-4 \
+        --zone "${ZONE}" \
+        --boot-disk-size 20 "${LABELS[@]}" &&\
+    shout "Created kyma-integration-test-${RANDOM_ID} in zone ${ZONE}" && break
+done || exit 1
```

Comment:
If it ends with failure, we should print out an error message.

Reply:
If it ends in a failure we will get the error msg from the gcloud command - the same that happened in the issue #112. Isn't that enough? Or would we want something like `shout "Machine could not been created in zone ${ZONE}` ?

Reply:
Nope, I meant if it cannot create a vm after all tries at the very end before exiting. Something like `Failed to create an instance after # of attempts. Exiting..`